pkg/scheduler/core/generic_scheduler.go

```
@@ -546,8 +546,7 @@ func (g *genericScheduler) findNodesThatFit(pluginContext *framework.PluginConte
 						extender, err)
 					continue
 				}
-
-				return []*v1.Node{}, FailedPredicateMap{}, framework.NodeToStatusMap{}, err
+	                        return filtered, failedPredicateMap, filteredNodesStatuses, nil
```

No offense, I think this should return an error